### PR TITLE
Fix/update references

### PR DESCRIPTION
**Pull request by digisomni**
Updates license headers and updates references to 'kasenvr' repo to 'vircadia' repo.

### CODE REVIEW

File: BUILD_ANDROID.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ The above code to suppress modules is not necessary, but will speed up the build
 
 ### Clone the repository
 
-`git clone https://github.com/kasenvr/project-athena.git`
+`git clone https://github.com/vircadia/project-athena.git`
```
Comment (ctrlaltdavid) [inline BUILD_ANDROID.md:69]:
```suggestion
`git clone https://github.com/vircadia/vircadia.git`
```
And elsewhere.

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

File: BUILD_ANDROID.md (hunk 1 above)
Comment (ctrlaltdavid) [reply BUILD_ANDROID.md]: There are other occurrences, too.

File: interface/resources/qml/hifi/dialogs/TabletAboutDialog.qml [hunk 2]
```diff
@@ -54,10 +54,10 @@ Rectangle {
             textFormat: Text.StyledText
             linkColor: "#00B4EF"
             color: "white"
-            text: "<a href=\"https://github.com/kasenvr/project-athena\">Vircadia Github</a>."
+            text: "<a href=\"https://github.com/vircadia/project-athena\">Vircadia Github</a>."
```
Comment (ctrlaltdavid) [inline interface/resources/qml/hifi/dialogs/TabletAboutDialog.qml:57]:
```suggestion
            text: "<a href=\"https://github.com/vircadia/vircadia\">Vircadia Github</a>."
```

File: interface/resources/qml/hifi/dialogs/TabletAboutDialog.qml [hunk 3]
```diff
@@ -54,10 +54,10 @@ Rectangle {
...
             size: 20
             onLinkActivated: {
-                About.openUrl("https:/github.com/kasenvr/project-athena");
+                About.openUrl("https:/github.com/vircadia/project-athena");
```
Comment (ctrlaltdavid) [inline interface/resources/qml/hifi/dialogs/TabletAboutDialog.qml:60]:
```suggestion
                About.openUrl("https:/github.com/vircadia/vircadia");
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR906/Vircadia-Alpha-PR906-2185fc0.tar_1634138c.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR906/Vircadia-Alpha-PR906-2185fc0_1634138c.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR906/Vircadia-Alpha-PR906-2185fc0.tar_1634138c.gz

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR906/Vircadia-Alpha-PR906-2185fc0_1634138c.dmg

**build (ubuntu-18.04, android)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR906/Vircadia-Quest-Alpha-PR906-2185fc0_1634138c.apk
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR906/Vircadia-Alpha-PR906-2185fc0_1634138c.apk

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR906/Vircadia-Alpha-PR906-301cee5_261c2e6c.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR906/Vircadia-Alpha-PR906-301cee5_261c2e6c.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR906/Vircadia-Alpha-PR906-301cee5.tar_261c2e6c.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR906/Vircadia-Alpha-PR906-301cee5.tar_261c2e6c.Z

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR906/Vircadia-Alpha-PR906-301cee5_261c2e6c.exe

**build (ubuntu-18.04, android)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR906/Vircadia-Quest-Alpha-PR906-301cee5_261c2e6c.apk
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR906/Vircadia-Alpha-PR906-301cee5_261c2e6c.apk

**Review by ctrlaltdavid — APPROVED** [review]

**Pull request merged**